setup.py

```diff
@@ -119,7 +119,7 @@
         "flaky",
         "responses>=0.7",
         "conllu==2.3.2",
-        "transformers>=2.4.0,<2.5.0",
+        "transformers>=2.6.0",
```

Comment:
Are you sure you don't want to keep an upper bound?

Reply:
Probably a good idea. Added in 3264dcc.